lib/ballerCard.dart

```diff
@@ -103,22 +95,40 @@ class LeaderBallerCard extends StatelessWidget {
             child: Image(image: AssetImage('assets/raysmall.png')),
           ),
           Expanded(
-            child: Text(name, 
-            style: TextStyle(
-              color: Colors.black,
-              fontFamily: 'Open Sans',
-              fontWeight: FontWeight.bold,
-              fontSize: 14.0,
-              
-            ), textAlign: TextAlign.center,),
+            child: Text(
+              name,
+              style: TextStyle(
+                color: Colors.black,
+                fontFamily: 'Open Sans',
+                fontWeight: FontWeight.bold,
+                fontSize: 14.0,
+              ),
+              textAlign: TextAlign.center,
+            ),
           ),
           Expanded(
-            child: Text(bp.toString() + " BP",textAlign: TextAlign.right,),
+            child: Text(
+              bp.toString() + " BP",
+              textAlign: TextAlign.right,
+            ),
           ),
-          Expanded(
-            child: Icon(MyFlutterApp.basketball_ball))
+          Expanded(child: Icon(MyFlutterApp.basketball_ball))
         ],
       ),
     );
   }
 }
+
+class User {
+  const User({
+    this.firstName,
+    this.lastName,
+    this.username,
+    this.bp,
```

Comment:
Does `bp` stand for `ballerPoints`? Can we just have `points` here for clarity and simplicity sake?